src/components/Settings/CoreSettings.tsx

```diff
@@ -217,9 +217,12 @@ export const CoreSettings = () => {
         commands. Make sure you trust all the users you give these permissions to.
       </ConfirmDialog>
       {openPortModal}
-      <div className="flex w-full flex-col gap-4 @4xl:flex-row">
-        <div className="w-[28rem]">
-          <h2 className="text-h2 font-bold tracking-medium"> Core Settings </h2>
+
+      <div className="flex w-full max-w-6xl flex-col gap-4 @4xl:flex-row px-6">
```

Comment:
can we match this with `profile.tsx` so all the setting pages look the same
also now these are full on pages, maybe we should move `CoreSettings` and the other settings pages to the pages folder.
Probably make a new folder `settings` under `pages`?

```suggestion
      <div className="flex w-full max-w-6xl flex-col gap-4 @4xl:flex-row px-6">
```